frontend/packages/console-shared/src/components/editor/BasicCodeEditor.scss

```diff
@@ -1,3 +1,11 @@
+.co-code-editor {
+  .monaco-editor {
+    // revert custom overrides
+    a.label-name { text-decoration-line: none; } // PF normalize.scss
```

Comment:
see https://github.com/patternfly/patternfly/pull/7458

Reply:
@logoff, you're supposed to be studying!  ;-)

Reply:
@logonoff, you're supposed to be studying!